src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.spec.ts

```diff
@@ -1,52 +1,51 @@
-import { AlmUserName } from './../../../pipes/alm-user-name.pipe';
 import {
-  async,
-  ComponentFixture,
-  fakeAsync,
-  inject,
-  TestBed,
-  tick
+async,
```

Comment:
I wonder what setting I have in my editor that removed these spaces, since I know I didn't do it manually.